packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -58,6 +58,8 @@ const trace = makeTracer('VM');
  *
  * @property {number}         numVaults        present count of vaults
  * @property {Amount<'nat'>}  totalCollateral  present sum of collateral across all vaults
+ * @property {Amount<'nat'>}  totalCollateralHeld collateral held as a result of not returning excess refunds
+ *                                                from AMM to owners of vaults liquidated with shortfalls
  * @property {Amount<'nat'>}  totalDebt        present sum of debt across all vaults
  *
  * @property {Amount<'nat'>}  totalCollateralSold       running sum of collateral sold in liquidation // totalCollateralSold
```

Comment:
This could read as a complement of the new value. An expression of their relationships would help. IIUK,

total collateral ever  = `totalCollateral` (now) + `totalCollateralSold` (ever)
`totalCollateralHeld` <<`totalCollateralSold` and can't be inferred from the other numbers

> This does add to the accounting records, if that's written up anywhere

Yep, this `MetricsNotification` object type. So the elaboration could go as a description of the type (which currently only has `@property` elements)

Reply:
"total collateral ever" includes a lot of collateral that was withdrawn by its owner, which we don't account for.
`totalCollateralHeld` was once a part of totalCollateral, but is not included in `totalCollateralSold`, as it was not sold.
At the end of a liquidation, `totalCollateral` is reduced by `collateralPre`, since all of that collateral was either sold, returned to the vault owner, or held by the VaultManager.
`collateralPost` can either be returned to the owner or held by the vaultManager, depending on the value of `shortfall`.

Does any of this need to be added to `MetricsNotification`, or described more completely in the comments here or in `liquidate()`?

Reply:
> I interpret the Documentation section as being about external user or developer documentation

Me too. The `MetricsNotification` type gets exported. Anyone consumer from the subscriber will see that `MetricsNotification` is the type of data coming out. With comments there they'll be able to understand what the properties of the object are counting.

> "total collateral ever" includes a lot of collateral that was withdrawn by its owner, which we don't account for.

Ah, there's no way to reconstruct that. Got it.

> Does any of this need to be added to MetricsNotification, or described more completely in the comments here or in liquidate()?

I think some of this should make it into MetricsNotification (either on the type itself or its properties). Though I think I've figured out what makes me uneasy: `totalCollateralHeld` uses the past-participle like the "running sums" but it's a "present value" (because it's in the seat, and it can go down). This doesn't match the scheme https://github.com/Agoric/agoric-sdk/blob/a28987519403147d505346a646839ffb3e18622e/packages/inter-protocol/src/vaultFactory/vaultManager.js#L55

Also now that I think about it "held" is ambiguous in the context of getting a metric notification because all the collateral is held.

A third issue is that it's not a "total" in the way that "totalCollateral" is; that's a total because it sums over all vaults. But this is one seat.

To solve those three, WDYT of `unsoldCollateral`?

Reply:
as discussed, switched to `retainedCollateral`.